cmd/linuxdb.yaml

```diff
@@ -122,3 +129,21 @@ distributions:
           help: Please install using your system's package manager
         - name: npm
           help: Please install using your system's package manager
+  alpine:
```

Comment:
Did you try to build one of the template apps in Alpine? I'm really curious as it uses libmusl.

Reply:
yeap, it works!

Reply:
```bash
Wails v0.17.11-pre - Initialising project

The name of the project (My Project): 
Project Name: My Project
The output binary name (my-project): 
Output binary Name: my-project
Project directory name (my-project): 
Project Directory: my-project
Please select a template:
  1: Angular - Angular 8 template (Requires node 10.8+)
  2: React JS - Create React App v3 template
  3: Vue2/Webpack Basic - A basic Vue2/WebPack4 template
  4: Vuetify Basic - Vuetify + Webpack
Please choose an option [1]: 3
Template: Vue2/Webpack Basic
✓ Generating project...
✓ Building project (this may take a while)...
Project 'My Project' built in directory 'my-project'!
```